app/models/gravatar.rb

```diff
@@ -0,0 +1,46 @@
+class Gravatar
+  GRAVATAR_ENDPOINT = 'https://www.gravatar.com/avatar/'.freeze
+
+  # A bogus Gravatar ID we can use to avoid leaking the email hash
+  # of users that prefer to not have their emails disclosed.
+  #
+  # @see https://en.gravatar.com/site/implement/images/#default-image
+  GRAVATAR_DEFAULT_ID = "00000000000000000000000000000000"
```

Comment:
Right now this will mean that everyone would have the same default image. Could we default it to perhaps `user_id % 5` or another operation/number so not all users will have the same image? 

eg. https://www.gravatar.com/avatar/1?d=retro would be different from https://www.gravatar.com/avatar/0?d=retro